### PR TITLE
create videoComponent for catch files to backend

### PR DESCRIPTION
**Pull request by JulienBenoit7**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,9 @@
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "husky": "^8.0.3"
+  },
+  "dependencies": {
+    "axios": "^1.4.0",
+    "mysql": "^2.18.1"
```
Comment (rocambille) [inline package.json]: uninstall these from root

File: frontend/src/components/VidéoComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          "Une erreur s'est produite lors de la récupération des vidéos :",
+          error
+        );
+      });
+  }, []);
+
+  return (
+    <div>
+      {videos.map((video) => (
+        <div key={video.id}>
```
Comment (rocambille) [inline frontend/src/components/VidéoComponent.jsx]: you can use a figure here

File: frontend/src/components/VidéoComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <p>{video.title}</p>
```
Comment (rocambille) [inline frontend/src/components/VidéoComponent.jsx]: figcaption ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/VidéoComponent.jsx
Comment (Justine-Merlin) [inline frontend/src/components/VidéoComponent.jsx:1]: No mark on letter in english, this file should be name `VideoComponent`

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rocambille) [reply package.json]: mysql is still installed :)

File: frontend/src/components/VidéoComponent.jsx (hunk 2 above)
Comment (rocambille) [reply frontend/src/components/VidéoComponent.jsx]: you used figcaption, but you forgot the figure here

File: frontend/src/images/Capture d'écran 2023-06-06 113504.png
Comment (Justine-Merlin) [inline frontend/src/images/Capture d'écran 2023-06-06 113504.png:1]: You should rename this file with a correct name and without whitespace.

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

**Pull request merged**